Dockerfile

```
@@ -128,11 +128,14 @@ WORKDIR /app
 COPY api /app/
 COPY .release-please-manifest.json /app/.versions.json
 
+ARG PROMETHEUS_MULTIPROC_DIR="/tmp/prometheus"
 ARG ACCESS_LOG_LOCATION="/dev/null"
-ENV ACCESS_LOG_LOCATION=${ACCESS_LOG_LOCATION} DJANGO_SETTINGS_MODULE=app.settings.production
+ENV ACCESS_LOG_LOCATION=${ACCESS_LOG_LOCATION} PROMETHEUS_MULTIPROC_DIR=${PROMETHEUS_MULTIPROC_DIR} DJANGO_SETTINGS_MODULE=app.settings.production
```

Minor formatting improvement suggestion(s). 

```suggestion
ENV ACCESS_LOG_LOCATION=${ACCESS_LOG_LOCATION} \
  PROMETHEUS_MULTIPROC_DIR=${PROMETHEUS_MULTIPROC_DIR} \
  DJANGO_SETTINGS_MODULE=app.settings.production
```

or 

```suggestion
ENV ACCESS_LOG_LOCATION=${ACCESS_LOG_LOCATION}
ENV PROMETHEUS_MULTIPROC_DIR=${PROMETHEUS_MULTIPROC_DIR}
ENV DJANGO_SETTINGS_MODULE=app.settings.production
```